main.py

```diff
@@ -858,7 +883,7 @@ def escape_appscript(text: str) -> str:
     # Print message in chunks respecting character limit
     chunk_size = MESSAGE_LIMIT - 6
     for i in range(0, len(appscript), chunk_size):
-        await message.channel.send("```{}```".format(appscript[i : i + chunk_size]))
+        await message.channel.send("```js\n{}```".format(appscript[i : i + chunk_size]))
```

Comment:
We could probably have busty post a link to https://script.google.com/home/projects/create as well for ease of use.

Reply:
I ended up doing this in https://github.com/anoadragon453/busty/pull/119/commits/ca3ffaf19e1847242aabaace193b796d1ea82f98. It looks like the following:

![image](https://user-images.githubusercontent.com/1342360/185118129-01e835db-092f-45a1-abc7-1cc996747f5b.png)